www/docs/src/content/docs/how-it-works/index.mdx

```diff
@@ -0,0 +1,79 @@
+- **Virtual Imports**: The StudioCMS Core integration adds several [Vite Virtual Imports](https://vitejs.dev/guide/api-plugin#virtual-modules-convention) packed with features that give you full control over your dynamic content.
+- **Logging**: The StudioCMS Core integration logs all changes to Astro Studio and can be enabled or disabled through a config option.
+- **Default Frontend Routes**: The StudioCMS Core integration adds default frontend routes to your Astro project to allow for easy access to the StudioCMS dashboard.
+- **Plugin System**: You can easily extend StudioCMS using its own plugin system to create additional functionality.
+
+#### Core Virtual Modules
+
+- **`virtual:studiocms/config`**: Provides the resolved configuration for the StudioCMS integration ecosystem.
+- **`virtual:studiocms/version`**: Provides the current version of the StudioCMS integration ecosystem.
+- **`virtual:studiocms/_nav`**: Provides the navigation mappings for the user-facing front-end.
+- **`virtual:studiocms/astromdremarkConfig`**: Provides the configuration for Astro Markdown support.
+- **`studiocms:components`**: Provides the front-end exported components for StudioCMS.
+- **`studiocms:helpers`**: Provides the front-end exported helper functions for StudioCMS.
+
+### StudioCMS: Dashboard
+
+#### Overview
+
+The StudioCMS Dashboard is a web-based interface that allows you to manage your AstroStudioCMS project. It provides a user-friendly interface for creating, editing, and deleting content for your Astro project. The StudioCMS Dashboard is built with [Astro](https://astro.build) and [DaisyUI](https://daisyui.com/) with [UnoCSS](https://unocss.dev).
+
+#### Features
+
+- **Dashboard**: The StudioCMS Dashboard provides a user-friendly interface for managing your AstroStudioCMS project.
+- **UnoCSS**: The StudioCMS Dashboard uses UnoCSS for styling and theming.
+- **DaisyUI**: The StudioCMS Dashboard uses DaisyUI for styling and theming.
+- **AuthConfig**: The StudioCMS Dashboard provides authentication configuration options.
+- **Astrolace**: The StudioCMS Dashboard is built using [Astrolace](https://github.com/matthiesenxyz/astrolace) a [Shoelace.style](https://shoelace.style) Astro Integration.
+- **DashboardRouteOverride**: The StudioCMS Dashboard provides a dashboard route override configuration option. Allowing you to change the default route for the StudioCMS Dashboard and API.
+
+#### Dashboard Virtual Modules
+
+- **`studiocms-dashboard:auth`**: The Virtual module that provides the authentication configuration for the StudioCMS Dashboard.
+- **`studiocms-dashboard:components`**: The Virtual module that provides the exported dashboard components for the StudioCMS Dashboard.
+- **`studiocms-dashboard:contentHelpers`**: The Virtual module that provides the exported content helper functions for the StudioCMS Dashboard.
+- **`studiocms-dashboard:routeMap`**: The Virtual module that provides the exported routes for the StudioCMS Dashboard.
```

Comment:
Not entirely happy with this rewrite, but NWTWWHB
```suggestion
- **`studiocms-dashboard:auth`**: Provides the authentication configuration for the dashboard.
- **`studiocms-dashboard:components`**: Provides dashboard components.
- **`studiocms-dashboard:contentHelpers`**: Provides content helper functions for the dashboard.
- **`studiocms-dashboard:routeMap`**: Provides the routes for the dashboard.
```

Reply:
This file was modified during your review